shs/update_api.sh

```
@@ -1,38 +1,22 @@
 #!/bin/bash
-set -e  # إيقاف التنفيذ عند حدوث أي خطأ
+cd $HOME
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)